doc/man1/openssl-verification-options.pod

```diff
@@ -426,6 +537,127 @@ This option has no effect and is retained for backward compatibility only.
 
 =back
 
+=head2 Certificate Extensions
+
+Options like B<-purpose> lead to checking the certificate extensions,
+which determine what the target certificate and intermediate CA certificates
+can be used for.
+
+=head3 Basic Constraints
+
+The basicConstraints extension CA flag is used to determine whether the
+certificate can be used as a CA. If the CA flag is true then it is a CA,
+if the CA flag is false then it is not a CA. B<All> CAs should have the
+CA flag set to true.
+
+If the basicConstraints extension is absent,
+which includes the case that it is an X.509v1 certificate,
+then the certificate is considered to be a "possible CA" and
+other extensions are checked according to the intended use of the certificate.
+The treatment of certificates without basicConstraints as a CA
+is presently supported, but may in the future be withdrawn.
+
+=head3 Key Usage
+
+If the keyUsage extension is present then additional restraints are
+made on the uses of the certificate. A CA certificate B<must> have the
+keyCertSign bit set if the keyUsage extension is present.
+
+=head3 Extended Key Usage
+
+The extKeyUsage (EKU) extension places additional restrictions on the
```

Comment:
This omits the fact that in some cases a CA' EKU must be a superset of any certificate it signs.

Reply:
> This omits the fact that in some cases a CA' EKU must be a superset of any certificate it signs.

Where to find details on that? Its this really implemented?
RFC 5280 just says:
> In general, this extension will appear only in end entity certificates.

Reply:
Notice the escape clause: "In general"

Again I repeat myself that RFC 5280 is not the final word on how the WebPKI works.  I commented earlier on this specific issue about EKU.

Reply:
I would focus on documenting what is currently implemented, not on RFC 5280 nor any other WebPKI.